plugin/tests/js/wizard/components/notice/__snapshots__/index.test.js.snap

```diff
@@ -8,9 +8,13 @@ exports[`Notice matches snapshot 1`] = `
   <div
     className="notice material-wizard__notice notice-error"
   >
-    <p>
-      There was an error
-    </p>
+    <p
+      dangerouslySetInnerHTML={
```

Comment:
Why does this need to be dangerous set? Is there another way to approach this?

Reply:
Lets sanitize

Reply:
@PatelUtkarsh was this missed? Should be sanitized or removed

Reply:
Hey @rodydavis, This snap test is being generated after sanitization which we've added here: https://github.com/material-components/material-design-for-wordpress/pull/165/files#diff-28d761d68fde552ff61fcfc790098f46fe998345dc66b22fec33f4faebe0653eR32.